### PR TITLE
Document grammar attributes

### PR DESCRIPTION
**Pull request by Kaiepi**
## The problem
Using attributes in grammars is undocumented.

## Solution provided
Document and provide an example using them.

### CODE REVIEW

File: doc/Language/grammars.pod6 [hunk 1]
```diff
@@ -288,7 +288,7 @@ the number for the currently parsed C<digit> digits.
     say Digifier.parse('255 435 777', actions => Devanagari.new).made;
     # OUTPUT: «(२५५ ४३५ ७७७)␤»
 
-=head2 Methods in Grammar
+=head2 Methods in Grammars
```
Comment (JJ) [inline doc/Language/grammars.pod6]: Can you please not use capitals here? Either you use C<Grammar>s or simply grammars.

**Comment by Kaiepi** [comment]
Updated

**Comment by Kaiepi** [comment]
I'll see if I can come up with a better example than this. I don't think this is very legible

**Comment by JJ** [comment]
@Kaiepi the example is correct, but what you are explaining, attributes and methods, take a very small part of the total real estate. Maybe a simpler grammar would be enough...

**Comment by Kaiepi** [comment]
Updated
I added more details since there was some stuff I was forgetting to mention, along with why attributes don't work in tokens. The example isn't much shorter, but should better illustrate how they behave

**Pull request merged**